### PR TITLE
Pos

### PR DESCRIPTION
**Pull request by Abdur-rahmaanJ**
Point of Sales modules. Removes products as transactions process. Base to be improved.

### CODE REVIEW

File: shopyo/static/css/b4vtabs.min.css [hunk 1]
```diff
@@ -0,0 +1 @@
...ex-direction:row;border-right:none;border-left:none;min-width:100%;border-bottom:1px solid #dee2e6;left:auto;margin-top:auto;padding-bottom:.25rem}.right-tabs.nav-tabs{flex-direction:row;border-right:none;border-left:none;min-width:100%;border-top:1px solid #dee2e6;right:auto;margin-top:auto;padding-top:.25rem}.sideways-tabs.left-tabs .nav-item,.sideways-tabs.right-tabs .nav-item{transform:none;height:auto;width:auto;margin-bottom:0}.sideways-tabs.left-tabs .nav-link,.sideways-tabs.right-tabs .nav-link,.nav-tabs .nav-link{width:8rem;text-align:center;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;border-radius:.25rem;margin:.25rem;border-top-color:transparent;border-right-color:transparent;border-bottom-color:transparent;border-left-color:transparent}.sideways-tabs.nav-tabs .nav-link:hover,.nav-tabs .nav-link:hover{border:1px solid #e9ecef;border-top-color:#e9ecef;border-right-color:#e9ecef;border-bottom-color:#e9ecef;border-left-color:#e9ecef}.sideways-tabs.nav-tabs .nav-link.active,.left-tabs .nav-link.active,.right-tabs .nav-link.active{border-top-color:#dee2e6;border-right-color:#dee2e6;border-bottom-color:#dee2e6;border-left-color:#dee2e6}}.tab-content{padding:1rem}
```
Comment (memoglr) [inline shopyo/static/css/b4vtabs.min.css:1]: this seems to be a single line code

File: shopyo/modules/pos/models.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+
+class Transaction(db.Model):
+    __tablename__ = "transactions"
+
+    id = db.Column(db.Integer, primary_key=True)
+    chashier_id = db.Column(db.Integer)
+
+    def add(self):
+        """Save category to the database"""
```
Comment (memoglr) [inline shopyo/modules/pos/models.py]: Add transaction to the database

File: shopyo/modules/pos/models.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        db.session.add(self)
+
+    def insert(self):
+        """Save category to the database"""
+        db.session.add(self)
+        db.session.commit()
+
+    def update(self):
```
Comment (memoglr) [inline shopyo/modules/pos/models.py:18]: Update transaction to the database

File: shopyo/modules/pos/models.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        """Update category"""
+        db.session.commit()
+
+    def delete(self):
+        """delete category"""
```
Comment (memoglr) [inline shopyo/modules/pos/models.py]: Delete transaction from the database

File: shopyo/modules/pos/models.py [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    __tablename__ = "transactions"
+
+    id = db.Column(db.Integer, primary_key=True)
+    chashier_id = db.Column(db.Integer)
+
+    def add(self):
+        """Save category to the database"""
+        db.session.add(self)
+
+    def insert(self):
```
Comment (memoglr) [inline shopyo/modules/pos/models.py:15]: This could be also Add transaction to the database?

File: shopyo/modules/product/models.py [hunk 6]
```diff
@@ -14,3 +19,23 @@ class Product(db.Model):
     category_name = db.Column(
         db.String(100), db.ForeignKey("category.name"), nullable=False
     )
+    transactions = db.relationship("Transaction", secondary=transaction_helpers, backref="products", cascade="all, delete")
+
+    def add(self):
+        """Save category to the database"""
```
Comment (memoglr) [inline shopyo/modules/product/models.py]: Save product to the database

File: shopyo/static/css/b4vtabs.min.css (hunk 1 above)
Comment (Abdur-rahmaanJ) [reply shopyo/static/css/b4vtabs.min.css:1]: minified file

**Pull request merged**